CONTRIBUTING.md

```diff
@@ -100,17 +100,34 @@ If you are a part of the SedTRAILS team and have write access to the SedTRAILS G
 
 1. Create a pull request, for an example, following the instructions [here](https://help.github.com/articles/creating-a-pull-request/).
```

Comment:
I spot a way to make this sentence sound better. I think we can use: `Create a pull request; for an example, follow the instructions... `